source/native/symbulation.cc

```diff
@@ -82,7 +82,7 @@ int main(int argc, char * argv[])
   //inject organisms
   for (size_t i = 0; i < POP_SIZE; i++){
     Host *new_org;
-    if (random_phen_host) new_org = new Host(random.GetDouble(-1, 1));
+    if (random_phen_host) new_org = new Host(random.GetDouble(-1, 0));
```

Comment:
This makes sense for Lysis development, but could be forgotten and turn into a bug much later. It should be implemented with a configuration option. Rolled back in 9f9ccca.